pkg/app/run_test.go

```diff
@@ -790,6 +790,16 @@ COMMANDS
         --server-side-encryption-kms-key-id=SERVER-SIDE-ENCRYPTION-KMS-KEY-ID
```

Comment:
I went through each documented logging endpoint checking which one's supported `compression_codec` and from what I can see is that the following log endpoints support it but aren't included in this PR. 

Is there a reason for this absence? 

- Cloudfiles
- DigitalOcean
- Kafka

**UPDATE**: I can see that Kafka appears to have slightly different compression values, which maybe why you've not included that one in this PR, but the other two (Cloudfiles and DigitalOcean appear to have the same values as what's defined in this PR).

Reply:
Good question. Kafka already has the compression support. Cloudfiles and DigitalOcean aren't included only because they weren't on the original list I had of our endpoints that were part of this addition. I'll need to add the compression support for these two to `go-fastly` before I can include them in this cli change.

Reply:
👍🏻